contracts/Genesis.sol

```diff
@@ -226,25 +254,85 @@ contract Genesis is ERC721Pausable, VRFConsumerBase, Ownable {
      * @param nonce nonce used to verify that the caller is allowed to mint
      * @param proof Proof to verify that the caller is allowed to mint
      */
-    function mintWhitelist(uint256 nonce, bytes32[] calldata proof) external payable whenNotPaused() onlyWhitelist(nonce, proof) {
-        require((tokenCounter.current() + WHITELIST_MINT_COUNT) < MAX_SUPPLY, "Not enough supply");
+    function mintWhitelist(uint256 nonce, bytes32[] calldata proof)
+        external
+        payable
+        whenNotPaused
+        seedGenerated
+        onlyWhitelist(nonce, proof)
+    {
+        require(
+            (tokenCounter.current() + WHITELIST_MINT_COUNT) < MAX_SUPPLY,
+            "Not enough supply"
+        );
         require(msg.value >= PRICE, "Not enough ETH");
-        require((addressToMintCount[msg.sender] + addressToRequestCount[msg.sender]) < WHITELIST_MINT_COUNT, "Already minted");
-        requestMint();
+        require(
+            addressToMintCount[msg.sender] < WHITELIST_MINT_COUNT,
+            "Already minted"
+        );
+        uint256 tokenId = tokenCounter.current();
+        mint(msg.sender, tokenId, generateRandomNumber(tokenId));
     }
 
     /**
-     * Requests a random number to Chainlink VRF for a given token id
-     * @param sender sender of the request 
-     * @param tokenId id of the token
-     * @return requestId id of the request sent to Chainlink
+     * Mint reserved gods
+     * @param to address to send the god to
+     * @param count number of gods to transfer
+     */
+    function mintReservedGods(address to, uint256 count) public onlyOwner {
+        require(
+            reservedGodsSupply() >= count + reservedGodsTransfered.current(),
```

Comment:
this used to be `require(reservedGodsSupply() >= count, "Not enough reserved gods left");`, pretty sure that was not valid right?

Reply:
yeah this will all change anyway....